src/System/Terminal/Widgets/Select.hs

```diff
@@ -34,24 +34,44 @@ instance Widget Select where
                     , " "
                     , t
                     ]
-         in Text.unlines $ s.prompt : (mkOption <$> s.options)
+            prompt = s.prompt <> ":"
```

Comment:
Bad rebase: this line shouldn't exist anymore.

Reply:
whoops!